frameworks/C++/silicon/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Install silicon
+DIR=`pwd`
+rm -fr silicon;
+git clone https://github.com/matt-42/silicon.git
+cd silicon;
+CXX=/usr/bin/g++-4.9 ./install.sh $IROOT
+
+# Install microhttpd
+cd $DIR
+wget http://mirror.ibcp.fr/pub/gnu/libmicrohttpd/libmicrohttpd-0.9.39.tar.gz
+tar xvf libmicrohttpd-0.9.39.tar.gz
```

Comment:
Instead of `wget` and `tar`, please use `fw_get` and `fw_untar` (defined [here](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/bash_functions.sh#L10)) instead. This allows us to have uniform cleanup and avoid other issues. It's a simple change: 

```
fw_get http://mirror.ibcp.fr/pub/gnu/libmicrohttpd/libmicrohttpd-0.9.39.tar.gz
fw_untar libmicrohttpd-0.9.39.tar.gz
```